src/database/bplus/mod.rs

```diff
@@ -0,0 +1,88 @@
+//! # Bplus Database [WIP]
+//!
+//! This module provides a database implementation backed by a
+//! persistent bplus tree.
+//!
+//! #### Authorship
```

Comment:
Feel free to leave it in if you want, but I kinda removed all of the authorship tags from all files (already in `dev`).

I first did it because I saw it in Classic, and I figured I should do it too. But there is `git blame` nowadays, so there is really no need. Still, feel free to leave it in if you want to, it's nice in its own right to be people-first in that regard.

Reply:
i dont really care for it either, i was just modelling after other files in `src/database/~`. also its not about individual contributions, but about project progress 😸